docs/core-concepts/route-params.md

```diff
@@ -0,0 +1,156 @@
+# Route Params
+
+When defining your route `path`, you can use a colon `:` to denote dynamic params.
+
+```ts
+import { 
+  Routes,
+} from 'kitbag/router'
+
+const routes = [
+  {
+    name: 'users',
+    path: '/users/:id',
+    component: ...
+  }
+] as const satisfies Routes
+```
+
+This means the route will expect 1+ extra string characters (anything but a forward slash `/`) in order to be considered a match. The value of these extra characters is captured and exposed in the `route.params`
+
+```ts
+const route = useRoute()
+
+route.params.id
+```
+
+When defined this way, these params are reactive strings. If you update a param, the url will be updated accordingly.
```

Comment:
I'm not sure if this will be true or not. But fine to leave in for now. `route` is currently `DeepReadonly`

Reply:
okay, I'll add an issue for us to reconcile this difference before going live

Reply:
https://github.com/kitbagjs/router/issues/81